### PR TITLE
Switch NL from v1beta1 to v1. Fixes #2747.

### LINKED ISSUE #2747: Update to Language v1
### What
Update the hand-written code to Language v1, which was publicly announced yesterday (11/15). The new GAPIC code is in a [PR](https://github.com/googleapis/api-client-staging/pull/104), which will be published to PyPI as soon as its approved.

### Why
Language v1beta1 has been replaced by v1 as of Tuesday 11/15. It has some additional features we want to support.

This should be done soon, so that we can update the public documentation and samples to reference the `google-cloud-language` library.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: language/google/cloud/language/document.py [hunk 1]
```diff
@@ -188,6 +188,27 @@ def analyze_sentiment(self):
...
+                  the API.
+        """
+        data = {
+            'document': self._to_dict(),
+            'encodingType': self.encoding,
+        }
+        api_response = self.client._connection.api_request(
+            method='POST', path='analyzeSyntax', data=data)
+        return [Token.from_api_repr(token)
+                for token in api_response['tokens']]
```
Comment (dhermes) [inline language/google/cloud/language/document.py]: Worried about a `KeyError`? Instead `api_response.get('tokens', ())`?

File: language/unit_tests/test_document.py [hunk 2]
```diff
@@ -308,13 +308,164 @@ def test_analyze_sentiment(self):
...
+                        'case': 'CASE_UNKNOWN',
+                        'form': 'FORM_UNKNOWN',
+                        'gender': 'GENDER_UNKNOWN',
+                        'mood': 'MOOD_UNKNOWN',
+                        'number': 'SINGULAR',
+                        'person': 'PERSON_UNKNOWN',
+                        'proper': 'PROPER',
+                        'reciprocity': 'RECIPROCITY_UNKNOWN',
+                        'tense': 'TENSE_UNKNOWN',
+                        'voice': 'VOICE_UNKNOWN',
```
Comment (dhermes) [inline language/unit_tests/test_document.py]: Is **all** of this content necessary in the unit test? It's almost certainly going to make `pylint` cry to its mother

Comment (jmdobry) [inline language/unit_tests/test_document.py]: Done.

File: language/google/cloud/language/document.py (hunk 1 above)
Comment (jmdobry) [inline language/google/cloud/language/document.py]: Done.

**Comment by tseaver** [comment]
LGTM

**Comment by jmdobry** [comment]
Is there a set release schedule? What's the ETA on getting a new version of `google-cloud-language` published?

**Comment by dhermes** [comment]
I'll be releasing this afternoon, thanks for checking in

**Pull request merged**